site/style_about.css

```diff
@@ -0,0 +1,18 @@
+
+body {
+	border:solid 5px;
+	padding-left: 1.5cm;
+	padding-right: 1.5cm;
+	padding-top: .5in;
+	padding-bottom: 2cm;
+	font-family: helvetica;} 
+
+nav {text-align: center}
```

Comment:
Make sure you are consistent with how you format your CSS and indent your rules on a new line.